attributes/default.rb

```diff
@@ -1,5 +1,5 @@
 default[:apiaxle][:environment]                            = 'development'
-default[:apiaxle][:version]                                = '1.12.34'
+default[:apiaxle][:version]                                = '1.12.35'
```

Comment:
might be good to use a version number that apiaxle will never use to avoid collision?